cl/search/views.py

```diff
@@ -441,7 +465,9 @@ def show_results(request: HttpRequest) -> HttpResponse:
                 )
 
             # But give it a fresh form for the advanced search section
-            render_dict.update({"search_form": SearchForm(request.GET)})
+            render_dict.update(
+                {"search_form": SearchForm(request.GET)}, request=request
```

Comment:
is this right?

I noticed the `searchForm` class accepts a `request` object now and you've been updating several lines to accommodate this change but this line is different, we're adding the request to the render_dict object instead.

Reply:
yeah, this was a mistake, fixed it. Thanks for noticing it.